pkgs/by-name/xd/xdg-terminal-exec/package.nix

```diff
@@ -0,0 +1,31 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+}:
+stdenv.mkDerivation {
+  pname = "xdg-terminal-exec";
+  version = "unstable-20231208";
```

Comment:
https://github.com/NixOS/nixpkgs/blob/0838486e9183ce39398f01b221766e68b50f405c/pkgs/README.md?plain=1#L353

Date format is wrong. And the new convention sadly doesn't explain what to do when there is *no* latest upstream version <sub>( :confused: )</sub>, but inventing some version `0`[`.0`[`.0`]] seems to be the way to go.

(remember to adjust version in commit message as well)